apps/auth/config.py

```
@@ -0,0 +1,30 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+if not os.getenv("MONGO_URI"):
+    raise ValueError(
+        "\033[31m[ERROR]\033[0m \033[107mMONGO_URI not set, please set it in the .env file\033[0m"
+    )
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)